crates/cairo-coverage/tests/helpers.rs

```diff
@@ -0,0 +1,49 @@
+use assert_fs::fixture::PathCopy;
+use assert_fs::TempDir;
+use snapbox::cmd::{cargo_bin, Command as SnapboxCommand, Command};
+use std::{env, fs};
+
+/// # Panics
+///
+/// This function may panic if:
+///
+/// * Creating a temporary directory fails.
+/// * Copying project files to the temporary directory fails.
+/// * Reading the directory for trace files fails.
+/// * Running the `cairo-coverage` command fails.
+/// * Generating the HTML report using `genhtml` fails.
+/// * Reading the generated `lcov` file fails.
+/// * The `lcov` file does not exist in the expected location.
```

Comment:
Shouldn't this be obvious from the outputted error? Perhaps we can `.context()` instead of unwrapping here